packages/kbn-handlebars/src/__jest__/test_bench.ts

```diff
@@ -200,6 +191,12 @@ class HandlebarsTestBench {
     return handlebarsEnv.compileAST(this.template, this.compileOptions);
   }
 
+  private execBeforeRender() {
+    if (this.options.beforeRender) {
+      this.options.beforeRender();
+    }
```

Comment:
__optional nit:__ can also be reduced to something like this:
```suggestion
    this.options.beforeRender?.();
```